blueprints/custom-post/blueprints.json

```
@@ -0,0 +1,31 @@
+{
+    "$schema": "https://playground.wordpress.net/blueprint-schema.json",
+    "meta": {
+        "title": "Custom Post Type: Books",
+        "description": "Blueprint that added a custom post type to playground",
+        "author": "bph",
+        "categories": ["Content", "CPT"]
+    },
+    "landingPage": "/wp-admin/",
+    "steps":[
+            {
+                "step": "login"
+            },
+            {
+                "step": "mkdir",
+                "path": "/wordpress/wp-content/plugins/books"
+            },
+            {
+                "step": "writeFile",
+                "path": "/wordpress/wp-content/plugins/books/books.php",
+                "data": {
+                    "resource": "url",
+                    "url": "https://raw.githubusercontent.com/adamziel/blueprints/trunk/custom-post/books.php"
```

Almost there, one last change – the path to `books.php` is not `custom-post/books.php` but `blueprints/custom-post/books.php`:

```suggestion
                    "url": "https://raw.githubusercontent.com/adamziel/blueprints/trunk/blueprints/custom-post/books.php"
```

---

Darn it. Thanks.